Client/Frontend/Login Management/LoginListViewController.swift

```diff
@@ -50,6 +50,7 @@ class LoginListViewController: SensitiveViewController {
     fileprivate var selectionButtonHeightConstraint: Constraint?
     fileprivate var selectedIndexPaths = [IndexPath]()
     fileprivate let tableView = UITableView()
+    fileprivate let breachAlertsManager = BreachAlertsManager()
```

Comment:
This `breachAlertsManager` can be removed from here for now as we aren't using it yet.